src/services/member/plugins/profile/schemas.ts

```diff
@@ -1,90 +1,66 @@
 import { S } from 'fluent-json-schema';
+import { StatusCodes } from 'http-status-codes';
+
+import { FastifySchema } from 'fastify';
 
 import { error, uuid } from '../../../../schemas/fluent-schema';
 import { partialMember } from '../../../item/fluent-schema';
 
-const sharedProporities = {
-  bio: { type: 'string' },
-  facebookID: { type: 'string' },
-  linkedinID: { type: 'string' },
-  twitterID: { type: 'string' },
-};
 const memberSharedSchema = S.object()
+  .additionalProperties(false)
   .prop('bio', S.string())
   .prop('facebookID', S.anyOf([S.string(), S.null()]))
   .prop('linkedinID', S.anyOf([S.string(), S.null()]))
   .prop('twitterID', S.anyOf([S.string(), S.null()]));
 
 export const profileMember = S.object()
-  .additionalProperties(false)
   .prop('id', uuid)
+  .prop('bio', S.string())
   .prop('visibility', S.boolean())
-  .prop('member', partialMember)
-  .prop('createdAt', S.raw({}))
-  .prop('updatedAt', S.raw({}))
-  .extend(memberSharedSchema);
+  .prop('facebookID', S.string())
+  .prop('linkedinID', S.string())
+  .prop('twitterID', S.string())
+  .prop('createdAt', S.string().format('date-time'))
+  .prop('updatedAt', S.string().format('date-time'))
+  .prop('member', partialMember);
 
 export const createProfile = {
-  response: {
-    201: profileMember,
-    '4xx': error,
-  },
   body: S.object()
-    .prop('visibility', S.boolean())
     .additionalProperties(false)
+    .prop('visibility', S.boolean())
+    // QUESTION: why is the bio always required ? Can't we only update a social link ?
```

Comment:
I think it can be optional, but maybe we want to enforce users to fill this field for a better look in library (I made it up, might not be consistent with all endpoints). I'm good with any decision.